server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,19 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+async function checkWorkerAvailability(workerTimeslotId) {
+    //Selects all from workerTimeslot table 
+    const queryStatement = `select * from ${Tables.workerTimeslot}  where worker_timeslot_id = ${workerTimeslotId}`;
+    const queryOutput = await db.any(queryStatement);
+    //check if workertimeSlot is present 
+    if (_.isEmpty(queryOutput)) return false;
+    //check if available 
+    const currentStatus = _.map(queryOutput, 'status');
+    if (currentStatus == 'available') return true;
```

Comment:
Hi Nancy,
Just a reminder to use the `available` value in the`Timeslot` constants file to ensure code consistency.

Reply:
Hi Amy,

I would like to clarify this comment: The file server/constants/timeslot-status.json contains 2 possible values which the status can take:
 "available": "available",
 "unavailable": "unavailable"

Could you please clarify what you'd like me to change? 
Thanks!

Reply:
Hi Nancy,
I was suggesting that the hard-coded value on line 32 be adjusted to `TimeslotStatus.available` to reduce repetition and limit hardcoding.

Reply:
Hi Amy,
Thank you for clarifying!
My Action: I have replaced the hardcoded value of 'available' with TimeslotStatus.available.

You can see this change in line 32 of server/routes/private.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
I added a comment below that provides further detail on this.